code/core/system/proxy-velodyne16/src/velodyne16Decoder.cpp

```diff
@@ -233,7 +263,6 @@ void Velodyne16Decoder::nextString(const string &payload) {
             position += 2;
```

Comment:
Yes, you are right. Now it is fixed. The format is distance+intensity. There is no need to shift intensity to higher bits.